opensearch-api/lib/opensearch/api/actions/cat/allocation.rb

```diff
@@ -33,8 +33,10 @@ module Actions
         # @option arguments [List] :node_id A comma-separated list of node IDs or names to limit the returned information
         # @option arguments [String] :format a short version of the Accept header, e.g. json, yaml
         # @option arguments [String] :bytes The unit in which to display byte values (options: b, k, kb, m, mb, g, gb, t, tb, p, pb)
-        # @option arguments [Boolean] :local Return local information, do not retrieve the state from master node (default: false)
+        # @option arguments [Boolean] :local Return local information, do not retrieve the state from cluster_manager node (default: false)
+        # <b>DEPRECATED:</b> Please use <tt>cluster_manager_timeout</tt> instead.
```

Comment:
This does not work for YARD. Consider adding it on the same line as the option you want to mark as deprecated. For example:
`# @option arguments [Time] :cluster_manager_timeout (DEPRECATED: use cluster_manager_timeout instead)  Explicit operation timeout for connection to cluster_manager node`

Reply:
Updated to follow this convention.